README.md

```diff
@@ -1,457 +1,76 @@
-    "issuer": "https://{{yourOktaDomain}}.com/oauth2/default",
-    "clientId": "{{yourClientId}}",
-    "clientSecret": "{{yourClientSecret}}",
-    "redirectUri": "http://localhost:3000/authorization-code/callback"
-  }
-}
-```
-
-## Front-end
-
-When you start this sample, the [AngularJS 1.x UI](https://github.com/okta/samples-js-angular-1) is copied into the `dist/` directory. More information about the AngularJS controllers and views are available in the [AngularJS project repository](https://github.com/okta/samples-js-angular-1/blob/master/README.md).
-
-### Login Redirect
+### `resource-server`
 
-With AngularJS, we include the template directive `ng-click` to begin the login process. When the link is clicked, it calls the `login()` function defined in `login-redirect.controller.js`. Let’s take a look at how the `OktaAuth` object is created.
+A typical resource-server requires a frontend and a backend application, so you will need to start each process:
 
-```javascript
-// login-redirect.controller.js
-
-class LoginRedirectController {
-   constructor(config) {
-    this.config = config;
-  }
-   $onInit() {
-    this.authClient = new OktaAuth({
-      url: this.config.oktaUrl,
-      issuer: this.config.issuer,
-      clientId: this.config.clientId,
-      redirectUri: this.config.redirectUri,
-      scopes: ['openid', 'email', 'profile'],
-    });
-  }
- 
-  login() {
-    this.authClient.token.getWithRedirect({ responseType: 'code' });
-  }
-}
+backend:
+```bash
+cd resource-server
+mvn -Dokta.oauth2.issuer=https://dev-123456.oktapreview.com/oauth2/default
 ```
 
-There are a number of different ways to construct the login redirect URL.
-
-1. Build the URL manually
-2. Use an OpenID Connect / OAuth 2.0 middleware library
-3. Use [AuthJS](http://developer.okta.com/code/javascript/okta_auth_sdk)
-
-In this sample, we use AuthJS to create the URL and perform the redirect. An `OktaAuth` object is instantiated with the configuration in `.samples.config.json`. When the `login()` function is called from the view, it calls the [`/authorize`](http://developer.okta.com/docs/api/resources/oauth2.html#authentication-request) endpoint to start the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1).
-
-You can read more about the `OktaAuth` configuration options here: [OpenID Connect with Okta AuthJS SDK](http://developer.okta.com/code/javascript/okta_auth_sdk#social-authentication-and-openid-connect).
-
-**Important:** When the authorization code is exchanged for an `access_token` and/or `id_token`, the tokens **must** be [validated](#validation). We'll cover that in a bit.
+front-end:
 
-### Custom Login Form
-To render the [Okta Sign-In Widget](http://developer.okta.com/code/javascript/okta_sign-in_widget), include a container element on the page for the widget to attach to:
+This project is just serving a static `index.html` so you WILL need to update the configuration in that file, see line 53:
```

Comment:
It'd be nice to provide the exact path to index.html in front-end directory, namely `front-end/src/main/resources/static/index.html`